src/os/kstore/KStore.cc

```diff
@@ -348,6 +296,8 @@ static int get_key_object(const string& key, ghobject_t *oid)
     return -10;
   }
 
+  r = 0;
```

Comment:
@shinobu-x What's use of taking assiging 0 to r & returning?
Why not delete & return 0?